src/components/CurriculumComponents/CurriculumDownloadView/CurriculumDownloadView.test.tsx

```diff
@@ -0,0 +1,27 @@
+// import CurriculumDownloadView from ".";
+
+// import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";
+
+describe("CurriculumDownloadView", () => {
```

Comment:
Not sure if this is supposed to be skipped!

Reply:
Thanks I need to write one more test for it all combined, I'll do that. All the components parts are tested elsewhere